config/accounts.go

```diff
@@ -21,6 +21,9 @@ type Account struct {
 	GDPR               AccountGDPR `mapstructure:"gdpr" json:"gdpr"`
 	DebugAllow         bool        `mapstructure:"debug_allow" json:"debug_allow"`
 	DefaultIntegration string      `mapstructure:"default_integration" json:"default_integration"`
+	DefaultLimit       int         `mapstructure:"default_limit" json:"default_limit"`
+	MaxLimit           int         `mapstructure:"max_limit" json:"max_limit"`
+	DefaultCoopSync    bool        `mapstructure:"default_coop_sync" json:"default_coop_sync"`
```

Comment:
I recommend creating an object structure to provide context and further extension points in the future, such as embedding these three within a "UserSync" object.

Reply:
I agree with @SyntaxNode. I'd just say that we should probably use `cookie_sync` as the wrapper object for these fields so as to stay consistent with naming in Java.
Ref: https://github.com/prebid/prebid-server-java/blob/master/src/main/java/org/prebid/server/settings/model/Account.java#L23